data/Tejasp2003.json

```diff
@@ -0,0 +1,20 @@
+{
+	"name": "Tejas patel",
+	"type": "personal",
+	"bio": "Pre-Final year student of B.Tech CE",
+	"links": [{
+			"name": "Follow me on Twitter",
+			"url": "https://twitter.com/Tweetsoftejas",
+			"icon": "FaTwitter"
+		},
+		{
+			"name": "Follow me on Github",
+			"url": "https://twitter.com/Tweetsoftejas",
+			"icon": "FaGithub"
+		}, {
+			"name": "Follow me on LinkedIn",
+			"url": "https://www.linkedin.com/in/tejas2003/",
+			"icon": "FaGithub"
```

Comment:
You have used github icon for the linkedin url, please change it

Reply:
Do i have to create a new pull request to make edits?

Reply:
No @Tejasp2003 You can push changes to the same PR.